### DISCUSSION TITLE
Where is the code of deployed contract ?

### DISCUSSION THREAD

**Discussion by GaloisField2718** [near/nearcore#6819] [Q&A] [answered]
Hello, 

I would like to know if there is a possibility to have the code and especially functions of a deployed contract for example this one : https://explorer.testnet.near.org/transactions/BWCvYLc5wtrkDPn7LQsi1AFvFT5fCMuSUw9nWGJdgMy

**Comment by frol** [comment, marked as answer]
SDK teams are defining contract metadata standard, but there is no way to recover the code from Wasm binary, so there will be other ways to achieve that in the future.

**Reply by GaloisField2718** [reply]
I find a command to have something more readable : `wasm-nm`. 
It's not a total decompilation of wasm file but it's more understandable.
